[IT] Introduction, Design Principles, Technical References, Tests, Etc Translations Block (#736)
italia/eidas-it-wallet-docs: docs/it/how-to-read-spec.rst

.. include:: ../common/common_definitions.rst

Come leggere le Specifiche
--------------------------

> Questa specifica è progettata per soddisfare i requisiti di molteplici stakeholder all'interno dell'ecosistema IT-Wallet. Ogni ruolo ha responsabilità e ambiti diversi e, quindi, diverse esigenze informative. Questa sezione fornisce percorsi di lettura personalizzati per aiutarti a navigare in modo efficiente verso i contenuti più rilevanti per gli obiettivi di implementazione.

Reviewer: ```suggestion
Questa specifica è progettata per soddisfare i requisiti di molteplici stakeholder all'interno dell'ecosistema IT-Wallet. Ogni ruolo ha responsabilità e ambiti diversi e diverse esigenze informative. Questa sezione fornisce percorsi di lettura personalizzati per aiutare il lettore a navigare in modo efficiente verso i contenuti più rilevanti per gli obiettivi di implementazione.
```

Reviewer: evitiamo il linguaggio personale.

Panoramica della Struttura delle Specifiche
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

La specifica è organizzata nelle seguenti sezioni principali:

**Sezione** :ref:`introduction:Introduzione`: 
  Stabilisce l'ambito e il linguaggio normativo per l'ecosistema IT-Wallet.

**Sezione** :ref:`design:Principi di design`: 

>   Fornisce principi di progettazione e requisiti di identità del marchio per l'ecosistema IT-Wallet.

Reviewer: ```suggestion
  Fornisce principi di progettazione e requisiti di identità visiva dell'ecosistema IT-Wallet.
```

**Sezione** :ref:`appendix:Appendice`: 
  Fornisce dettagli tecnici supplementari, modelli di implementazione e framework di test, inclusa la gestione delle istanze di applicazioni mobili, specifiche di integrazione della piattaforma nazionale e matrici di test complete per la validazione dell'ecosistema.

Percorsi di Lettura per Ruolo
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Prima di immergersi nelle sezioni specifiche per ruolo, tutti i lettori dovrebbero familiarizzare con i concetti fondamentali delineati nelle Sezioni :ref:`introduction:Introduzione`, :ref:`architecture-overview:Panoramica dell'Architettura` e :ref:`trust:L'Infrastruttura di Trust`, che stabiliscono il vocabolario comune, i principi di progettazione e l'infrastruttura di fiducia che forniscono il quadro sottostante per l'intero ecosistema.

Fornitore di Wallet
"""""""""""""""""""

I lettori che implementano o gestiscono una Soluzione di **Fornitore di Wallet** dovrebbero concentrarsi sulla comprensione di come gestire in modo sicuro le Credenziali Elettroniche e facilitare le interazioni dell'Utente con altri partecipanti all'ecosistema.

**Sezioni essenziali:**

> * **Sezione** :ref:`wallet-solution:Soluzione Wallet`: Requisiti completi di implementazione del wallet, componenti e processi di interazione.

Reviewer: ```suggestion
* **Sezione** :ref:`wallet-solution:Soluzione Wallet`: Requisiti completi di implementazione del Wallet, componenti e processi di interazione.
```

* **Sezione** :ref:`digital-credential-management:Gestione delle Credenziali Digitali`: Modelli di dati, formati e gestione del ciclo di vita delle Credenziali Elettroniche.
* **Sezione** :ref:`digital-credential-flows:Flussi delle Credenziali Elettroniche`: Flussi di emissione e presentazione per le Credenziali Elettroniche.
* **Sezione** :ref:`wallet-provider-endpoint:Endpoint del Fornitore di Wallet`: Specifiche API per la federazione e specifiche per IT-Wallet.
* **Sezione** :ref:`algorithms:Algoritmi Crittografici`: Requisiti di implementazione per sicurezza e crittografia.
* **Sezione** :ref:`mobile-application-instance:Istanza dell'Applicazione Mobile`: Modelli di implementazione specifici per dispositivi mobili.
* **Sezione** :ref:`e-service-pdnd:e-Service PDND`: Integrazione con la Piattaforma Nazionale di Interoperabilità dei Dati.

**Sezioni secondarie:**

> * **Sezione** :ref:`credential-issuer-solution:Soluzione del Fornitore di Credenziale`: Comprensione delle interazioni e dei requisiti del fornitore di credenziali.

Reviewer: ```suggestion
* **Sezione** :ref:`credential-issuer-solution:Soluzione del Fornitore di Credenziale`: Comprensione delle interazioni e dei requisiti del Fornitore di Credenziali.
```

Relying Party
"""""""""""""

I lettori interessati all'implementazione o alla gestione di una Soluzione di **Relying Party** per accettare e verificare le Credenziali Elettroniche dovrebbero concentrarsi sui protocolli di presentazione e sui meccanismi di verifica.

**Sezioni essenziali:**

* **Sezione** :ref:`relying-party-solution:Soluzione di Relying Party`: Requisiti completi di implementazione del verificatore e Configurazione dell'Entità.
* **Sezione** :ref:`digital-credential-management:Gestione delle Credenziali Digitali`: Comprensione dei formati delle Credenziali Elettroniche e verifica della validità.
* **Sezione** :ref:`credential-presentation:Presentazione della Credenziale Digitale`: Implementazione del flusso di presentazione sia per scenari remoti che di prossimità.
* **Sezione** :ref:`relying-party-endpoint:Endpoint della Relying Party`: Specifiche API relative alla federazione e alla verifica.
* **Sezione** :ref:`algorithms:Algoritmi Crittografici`: Requisiti della suite crittografica.

**Sezioni secondarie:**

> * **Sezione** :ref:`wallet-solution:Soluzione Wallet`: Comprensione delle interazioni dell'Istanza del Wallet e dei protocolli di presentazione

Reviewer: ```suggestion
* **Sezione** :ref:`wallet-solution:Soluzione Wallet`: Comprensione delle interazioni dell'Istanza del Wallet e dei protocolli di presentazione.
```
